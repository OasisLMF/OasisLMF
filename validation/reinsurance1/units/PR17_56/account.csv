PortNumber,AccNumber,PolNumber,PolPerilsCovered,AccCurrency,LayerNumber,LayerParticipation,LayerLimit,LayerAttachment,PolPeril,PolDed6All,PolMinDed6All,PolMaxDed6All,PolLimit6All,OEDVersion
OEDTest,PR17_05,PR17_05_1,AA1,CAD,1,1,0,0,AA1,0,0,0,0,2.0.0
OEDTest,PR17_06,PR17_06_1,AA1,CAD,1,0.2,0,0,AA1,0,0,0,0,2.0.0
