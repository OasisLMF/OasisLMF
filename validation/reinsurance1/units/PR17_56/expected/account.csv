PortNumber,AccNumber,PolNumber,PolPerilsCovered,AccCurrency,LayerNumber,LayerParticipation,LayerLimit,LayerAttachment,PolPeril,PolDed6All,PolMinDed6All,PolMaxDed6All,PolLimit6All,CondPeril,CondNumber,CondMinDed6All,CondLimit6All
OEDTest,PR17_05,PR17_05_1,OO1,CAD,1,1.00,.00,.00,OO1,.00,.00,.00,.00,,,,
OEDTest,PR17_06,PR17_06_1,OO1,CAD,1,.20,.00,.00,OO1,.00,.00,.00,.00,,,,
OEDTest,SS01_04,SS01_04_1,OO1,CAD,1,1.00,.00,.00,OO1,.00,.00,.00,.00,,,,
